datacatalog/cloud-client/src/main/java/com/example/datacatalog/CreateFilesetEntry.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright 2019 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.datacatalog;
+
+// [START create_fileset_entry]
+
+import com.google.cloud.datacatalog.ColumnSchema;
+import com.google.cloud.datacatalog.CreateEntryGroupRequest;
+import com.google.cloud.datacatalog.CreateEntryRequest;
+import com.google.cloud.datacatalog.Entry;
+import com.google.cloud.datacatalog.EntryGroup;
+import com.google.cloud.datacatalog.EntryGroupName;
+import com.google.cloud.datacatalog.EntryName;
+import com.google.cloud.datacatalog.EntryType;
+import com.google.cloud.datacatalog.GcsFilesetSpec;
+import com.google.cloud.datacatalog.LocationName;
+import com.google.cloud.datacatalog.Schema;
+
+import com.google.cloud.datacatalog.v1beta1.DataCatalogClient;
+
+public class CreateFilesetEntry {
+
+  /**
+   * Create Fileset Entry
+   *
+   * @param projectId    The project ID to which the fileset belongs, e.g. 'my-project'
+   * @param entryGroupId The Entry Group ID to which the fileset belongs,
+   *                     e.g. 'fileset_entry_group'
+   * @param entryId      The Entry ID for the fileset, e.g. 'fileset_entry_id'
+   */
+  public static void createEntry(String projectId, String entryGroupId, String entryId) {
+
+    // -------------------------------
+    // Currently, Data Catalog stores metadata in the
```

Comment:
This doesn't need a line break, and the header lines (`// ----`) feel unnecessary.

Reply:
Removed.